robottelo/ui/base.py

```diff
@@ -596,6 +595,16 @@ def perform_action_chain_move(self, locator):
             )
         ActionChains(self.browser).move_to_element(element).perform()
```

Comment:
I don't mind to add it, but no failures for that part for long-long time, so not sure if we need to add redundant waiter